src/components/MultipieceModal/MultipieceModal.js

```diff
@@ -12,19 +15,26 @@ import {
 } from '@folio/stripes/components';
 
 const MultipieceModal = (props) => {
-  const { item, onClose, onConfirm } = props;
+  const {
+    intl: { formatMessage },
```

Comment:
Could you please add `intl` in `propTypes`?

Reply:
done